src/server/json_family.cc

```diff
@@ -65,6 +65,17 @@ inline void Evaluate(const json::Path& expr, const JsonType& obj, ExprCallback c
   });
 }
 
+inline JsonType Evaluate(const JsonExpression& expr, const JsonType& obj) {
```

Comment:
a special variant of Evaluate that does not accept a callback and instead returns all the hits as a json array.